force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -82,14 +101,23 @@ public with sharing class ClassesObjectsMethods {
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
 
+        List<String> splitSentenceIntoWords = sentence.split(' ');
+
         // Create a set to store the unique words
+        
+        Set<String> storeUniqueWords = new Set<String>(splitSentenceIntoWords); 
 
         // Add each word to the set
 
+        for(String addWordToSet : splitSentenceIntoWords) {
+            storeUniqueWords.add(addWordToSet);
+        }
+
         // Join the unique words back into a sentence
+        String newUniqueSentence = String.join(new List<String>(storeUniqueWords),' ');
 
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return newUniqueSentence; // Replace null with the variable you used to store the result
```

Comment:
You made this harder than it needed to be. When you passed your list into the Set constructor any duplicates were removed. No need to iterate over and add them a second time.